Add ViteConf 2024 to Calendar (#2530)
mainmatter/mainmatter.com: src/calendar/2024-10-03-vite-conf.md

---
title: "ViteConf"
image: "/assets/images/calendar/2024-10-03-viteconf-24/viteconf-logo.png"
location: Online
url: https://viteconf.org/
kind: conference
color: purple
hero:
  image: "/assets/images/hero/viteconf-24.png"
  imageAlt: "ViteConf pyramid."
---

> ViteConf 2024 will be held online on October 3rd and 4th. Mainmatter’s Senior Developer, Chris Manson, will be presenting a talk titled “Ember’s Journey to Build with Vite”. This session will cover how Ember.js developers can now use Vite to build their applications, offering a new approach for Ember’s development workflow.

Reviewer: ```suggestion
ViteConf 2024 will be held online on October 3rd and 4th. Mainmatter’s Chris Manson, will present “Ember’s Journey to Build with Vite”. His talk will cover how Ember.js developers can now, after the successful Embroider initiative, use Vite to build their applications, offering a new approach for Ember development.
```
